src/components/ApiKeyAuth/ApiKeyAuthFlow/LandingContent.tsx

```diff
@@ -41,7 +41,7 @@ export function LandingContent({
     <ApiKeyAuthCardLayout>
       <FormControl>
         <FormLabel my="6">
-          <Heading as="h4" size="md">{`Set up ${capitalize(provider)} integration`}</Heading>
+          <Heading as="h4" size="md">{`Set up ${providerName} integration`}</Heading>
```

Comment:
is providerName guaranteed or should we fall back to the capitalize provider?

Reply:
`providerName` includes the logic to fallback to `capitalize(provider)` see: https://github.com/amp-labs/react/pull/473/files#diff-adac1f48e166b537f09a13bb4faa2fb2f865a822cd93a9891504e1961a2e0e3fL37

Reply:
I see thanks for clarifying.